coderedcms/project_template/project_name/settings/base.py

```diff
@@ -181,7 +176,7 @@
 
 # Wagtail settings
 
-WAGTAIL_SITE_NAME = '{{ sitename }}'
+WAGTAIL_SITE_NAME = "{{ sitename }}"
```

Comment:
You can completely undo your changes to this file... none of these changes look like they were needed for flake8, it will reduce conflicts.